src/commands/utils/graph.ts

```diff
@@ -0,0 +1,230 @@
+import { AuthenticationProvider, Client } from "@microsoft/microsoft-graph-client";
```

Comment:
💡 Question only please: Is this completely new class file or inspired from any external implementation?

Reply:
This class was inspired from Peter Bomber's previous work.